### PR TITLE
Featurelist

### PR DESCRIPTION
**Pull request by jlatour3**

### CODE REVIEW

File: frontend/src/components/FeatureList.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      });
+  }, []);
+  return features;
+};
+
+const FeatureList = () => {
+  const features = useFrequentFeatures();
+  return (
+    <ItemList>
+      <p style={{ color: '#A9A7A7' }}>Frequent Filters </p>
```
Comment (Arthelon) [inline frontend/src/components/FeatureList.js]: add left align

File: frontend/src/components/FeatureList.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      <TagContainer>
+        {features.map(feature => {
+          return <Feature name={feature.name} />;
```
Comment (Arthelon) [inline frontend/src/components/FeatureList.js]: When you render lists of items in React, you need to add a unique `key` prop to uniquely identify each List item. [Resource](https://reactjs.org/docs/lists-and-keys.html)

Comment (jlatour3) [reply frontend/src/components/FeatureList.js]: there is unique props in Feature:
const Feature = props => {
  const { name } = props;

**Comment by Arthelon** [comment]
When Feature tag is selected, that feature should be added as a selected feature in the `FeatureSelect` component

**Pull request merged**